force-app/main/default/lwc/templateTableDetails/templateTableDetails.js-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+	<apiVersion>57.0</apiVersion>
+	<isExposed>true</isExposed>
+	<targets>
+		<target>lightning__RecordPage</target>
```

Comment:
Please remove these targets since these are not necessary, they are nested inside the parent component itself.